posts/an-introduction-to-dqm.md

```diff
@@ -0,0 +1,63 @@
+
+### Data Lineages
+A data lineage is a record of your data's origins and where it moves over time. The ability to track, manage, and view your data's lineage can simplify tracking errors back to the data source. With a data lineage in place, it is easy to perform root cause analysis while investigating data issues.
+
+## Completeness and Comprehensiveness
+Data is considered complete or comprehensive when, according to the requirements, all of the data required for a particular use case is present and available to be used. Completeness is usually measured as a percentage.
+
+In a healthcare dataset, completeness would be defined as containing personal details, insurance information, and dates for prior visits. An address or phone number may also be collected, but are not mandatory. The percentage of completeness here would decrease with the absence of each critical piece of information.
+
+## Integrity
+Integrity measures the accuracy and consistency of your data over its lifecycle.  Each time data is replicated or transferred it should remain unaltered between operations.
+
+Maintaining data integrity is essential for several reasons. Data Integrity ensures recoverability, search-ability, traceability to the origin and connectivity. Improving the integrity of a dataset will increase your system's stability and performance while simultaneously improving reusability and maintainability.
+
+Data security is one of many measures that can be employed to maintain data integrity.  If changes made to data are a result of unauthorized access, this is a failure to both your data integrity efforts and your data security measures. Depending on the dataset, an integrity violation could result in the loss of a critical datastore or loss of life due to the failure of a life-critical system.
+
+## Timeliness
+Timeliness references whether the information is available when expected and needed. Timeliness means different things for different systems. A hotel booking website requires to-the-second updates, so prospective customers can see what rooms are available or unavailable in a hotel. For a billing or transaction management system, however, nightly processing could suffice.
+
+For large systems that rely on stream processing, timeliness is even more critical. The amount of data generated during peak periods can be considerably larger than the amount generated at low-activity periods. A stream processing platform needs to be adaptive so that it can scale according to the fluctuating demands.
+
+---
+
+How do you go about ensuring your data is accurate, consistent, complete, and timely? There are many ways to do so, dependent on your use case and particular needs. One way is to use the Audit, Balance, and Control (ABC) Framework. We'll be addressing this framework and how the three components ensure accuracy, consistency, completeness, timeliness, and integrity in a following blog.
```

Comment:
Any potential to conclude with a tie-in to our Data Practice, perhaps?  Just thinking in text here, lol